functions/auth.bash

```diff
@@ -44,15 +44,15 @@ change_password() {
     while [[ -z $pass ]]; do
       if ! pass1="$(whiptail --title "Authentication Setup" --passwordbox "\\nEnter a new password:" 9 80 3>&1 1>&2 2>&3)"; then echo "CANCELED"; return 0; fi
       if ! pass2="$(whiptail --title "Authentication Setup" --passwordbox "\\nPlease confirm the password:" 9 80 3>&1 1>&2 2>&3)"; then echo "CANCELED"; return 0; fi
-      if [[ $pass1 == "$pass2" ]] && [[ ${#pass1} -ge 10 ]] && [[ ${#pass2} -ge 10 ]]; then
+      if [[ $pass1 == "$pass2" ]] && [[ ${#pass1} -ge 8 ]] && [[ ${#pass2} -ge 8 ]]; then
```

Comment:
Why change to 8?

Reply:
I'm no fan of lengthy passwords. Being forced to invent one makes you use passes you can remember (hence way more insecure).
Had a discourse on that with @ThomDietrich a while back and he didn't mind changing it down to 8. I thought I had already done that back then now saw I didn't. If you believe it's inherently more secure to stick with 10 we can do.

Reply:
I have no real issue with it, I was just wondering.